src/Zygote.jl

```diff
@@ -12,7 +12,7 @@ using MacroTools, Requires
 using MacroTools: @forward
 
 import Distributed: pmap, CachingPool, workers
-export Params, gradient, pullback, pushforward, @code_adjoint
+export Params, gradient, jacobian, pullback, pushforward, @code_adjoint
```

Comment:
we could export hessian as well if we feel confident enough

Reply:
That would be fine by me, too.

Reply:
```suggestion
export Params, gradient, jacobian, hessian, pullback, pushforward, @code_adjoint
```
... seems consistent